libbeat/cfgfile/cfgfile.go

```diff
@@ -17,7 +17,7 @@ var testConfig *bool
 func init() {
 	// The default config cannot include the beat name as it is not initialised when this
 	// function is called, but see ChangeDefaultCfgfileFlag
-	configfile = flag.String("c", "/etc/beat/beat.yml", "Configuration file")
+	configfile = flag.String("c", "beat.yml", "Configuration file")
```

Comment:
That is a tricky one and we haven't found a solution yet. I don't think we should invest more time into this at the moment.